setup.py

```diff
@@ -27,8 +27,10 @@
 http://docs.python.org/2/distutils/sourcedist.html
 
 """
-import os
 from setuptools import setup, find_packages
+from sys import version_info
+import os
```

Comment:
The import statements looked all jumbled up to me, so I sorted them alphabetically. Will change as you prefer, though it seems a bit weird to have i before f :)